src/GAP.jl

```diff
@@ -119,8 +119,9 @@ function __init__()
     ## With Julia 1.2, this will be possible and this hack could then be replaced with the
     ## corresponding function call in in ccalls.jl (func::GapObj)(...)
     MPtr = Base.MainInclude.eval(:(ForeignGAP.MPtr))
+    global call_gap_func
     Base.MainInclude.eval(:(
-        (func::$MPtr)(args...; kwargs...) = GAP.call_gap_func(func, args...; kwargs...)
+        (func::$MPtr)(args...; kwargs...) = $(call_gap_func)(func, args...; kwargs...)
```

Comment:
So it doesn't work like this, without the `global` above?
```suggestion
        (func::$MPtr)(args...; kwargs...) = $(GAP.call_gap_func)(func, args...; kwargs...)
```
I just tested that locally, and it seemed to work fine? If there is no reason, let's use mine. If there is a reason, let's add a comment explaining it?

Reply:
@sebasguts am I missing something?

Reply:
@fingolfin I am not completely sure if this also works when GAP is loaded as a dependency of another module. Will test, and then change it accordingly (either comment or your version).

Reply:
Tested, and your approach is valid. I updated the PR.